src/pages/HomePage.tsx

```
@@ -1,10 +1,184 @@
+  </div>
+  </div>
+  <div>
+  </div>
+
+  {/* BITCOIN CARD */}
+ <div className="grid grid-cols-4 gap-7 py-6">
+   {cards && cards.map((card)=>(
+    <div key={card.id} >
+      <Card card={card}/>
+    </div>
+   ))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)